app/src/main/kotlin/io/homeassistant/companion/android/settings/sensor/views/SensorListView.kt

```diff
@@ -34,7 +34,7 @@ fun SensorListView(
     onSensorClicked: (String) -> Unit
 ) {
     LazyColumn {
-        viewModel.allSensors.forEach { (manager, currentSensors) ->
+        viewModel.allSensors.filter { it.value.isNotEmpty() }.forEach { (manager, currentSensors) ->
```

Comment:
Merge conflict

Reply:
It was not showing because the merge would't conflict because the change is exactly the same :)